src/generic/NumberField.jl

```diff
@@ -36,7 +38,7 @@ end
 #
 ###############################################################################
 
-function number_field(a::Poly{Rational{BigInt}}, s::Symbol, t = "\$"; cached::Bool=true)
+function number_field(a::Poly{Rational{BigInt}}, ::VarName; cached::Bool=true)
```

Comment:
So no code ever tries to pass `t`?

Reply:
Right, and as of #1485 this code also should not be used anywhere else.